Theme/ElegantFin-theme-nightly.css

```diff
@@ -1139,3 +1139,8 @@ div[data-role=controlgroup] a.ui-btn-active {
     border-radius: 8px;
     border-color: var(--borderColor);
 }
+
+.sectionTitleTextButton:not(.padded-left) {
+    margin: 0 -0.5em !important;
+    padding: 0 0.5em !important;
+}
```

Comment:
This is not necessary now because of 44466c16ff587961cc427d0e8173db5afb516fec, right?

Reply:
Correct, I thought I removed that